src/vivarium_gates_mncnh/components/children.py

```diff
@@ -0,0 +1,130 @@
+from typing import List, Tuple
+
+import numpy as np
+import pandas as pd
+from vivarium import Component
+from vivarium.framework.engine import Builder
+
+from vivarium_gates_mncnh.constants import data_keys, data_values, models
+
+
+class NewChildren(Component):
+    ##############
+    # Properties #
+    ##############
+
+    @property
+    def sub_components(self) -> List[str]:
+        return [self.lbwsg]
+
+    def __init__(self):
+        super().__init__()
+        self.lbwsg = LBWSGDistribution()
+
+    def setup(self, builder: Builder):
+        self.randomness = builder.randomness.get_stream(self.name)
+        self.male_sex_percentage = data_values.INFANT_MALE_PERCENTAGES[
+            builder.data.load(data_keys.POPULATION.LOCATION)
+        ]
+
+    def empty(self, index: pd.Index) -> pd.DataFrame:
+        return pd.DataFrame(
+            {
+                "sex_of_child": data_values.PREGNANCY_OUTCOMES.INVALID_OUTCOME,
+                "birth_weight": np.nan,
+                "gestational_age": np.nan,
+            },
+            index=index,
+        )
+
+    def generate_children(self, index: pd.Index) -> pd.DataFrame:
+        sex_of_child = self.randomness.choice(
+            index,
+            choices=["Male", "Female"],
+            p=[self.male_sex_percentage, 1 - self.male_sex_percentage],
+            additional_key="sex_of_child",
+        )
+        lbwsg = self.lbwsg(sex_of_child)
+        return pd.DataFrame(
+            {
+                "sex_of_child": sex_of_child,
+                "birth_weight": lbwsg["birth_weight"],
+                "gestational_age": lbwsg["gestational_age"],
+            },
+            index=index,
+        )
+
+
+class LBWSGDistribution(Component):
+    def setup(self, builder: Builder):
+        self.randomness = builder.randomness.get_stream(self.name)
+        self.exposure = builder.data.load(data_keys.LBWSG.EXPOSURE).set_index("sex")
+        self.category_intervals = self._get_category_intervals(builder)
+
+    def __call__(self, newborn_sex: pd.Series):
```

Comment:
Nit: I think it'd be better to have this be an explicit method called something like `get_exposure()` rather than having the component be a callable itself.